data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Works of Philo Judaeus</title>
```

Comment:
In the title of the individual work files we put the title of the work itself, in this case the English title above "On the Creation of the World." I chose the shorter title listed in the TOC for this volume rather than the lengthier title "A TREATISE THE ACCOUNT OF ON OF THE WORLD, THE CREATION AS GIVEN BY MOSES."

Reply:
done